content/guides/integrations/metaflow.md

```diff
@@ -34,6 +34,11 @@ For a more streamlined approach, you can generate an API key by going directly t
 
 To install the `wandb` library locally and log in:
 
+{{% alert %}}
+For wandb versions <= 0.19.8, install `fastcore<1.8.0` instead of `plum-dispatch`:
```

Comment:
```suggestion
For `wandb` version 0.19.8 or below, install `fastcore` version 1.8.0 or below (`fastcore<1.8.0`) instead of `plum-dispatch`.
```
This might be more readable in the prose, while also giving the correct syntax. Also, the alert doesn't have any more contents, so we should end the sentence with a period.